sdk/core/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# release 6.6.0 (2022-09-21)
+* Added `origin` to `AnchorDidInput`
+# release 6.5.2 (2022-09-21)
```

Comment:
Added this, has been missed for https://github.com/affinityproject/affinidi-core-sdk/pull/242